app/controllers/comments_controller.rb

```diff
@@ -1,27 +1,44 @@
 class CommentsController < ApplicationController
+  # The load_and_authorize_resource method automatically loads the resource (model) based on the controller's actions
```

Comment:
- [ ] Kindly import the `load_and_authorize_resource` method here, instead of leaving this comment